spec/examples/ex4-overflow.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<tt xmlns="http://www.w3.org/ns/ttml" xmlns:tts="http://www.w3.org/ns/ttml#styling" xml:lang="en" tts:extent="320px 240px">
+  <head>
+    <styling>
+      <style xml:id="s1" tts:backgroundColor="black" tts:wrapOption="noWrap" tts:padding="6px"/>
```

Comment:
Why put `tts:wrapOption` in the region style? It seems inconsistent to do this but put `tts:fontFamily` and `tts:fontSize` in the p style - those three should probably all go on the same style, I don't mind which one.